### ISSUE TITLE
Trouble performing Deleteuser after promote user and logging in as promoted user

### ISSUE THREAD

**Issue by rkakadia9855** [QihanLu/CS419#5]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/61147225/144748707-2ddc32a0-4643-4cfe-acf4-d765a7c7837a.png">

**Comment by QihanLu** [comment]
Hi, 
     According to the readme file, it should be Delete instead of Deleteuser. Sorry for the confusion in the prompt.   
However, we don't know if we should consider it as a programming bug and maybe we should let TA to make a decision.

**Comment by rkakadia9855** [comment]
I mean it is technically a bug right because the user of the application will see the prompt and follow the instruction. But I agree on letting the TA decide

**Comment by symxmyz233** [comment]
Yeah, it is acutally **a small bug**, developer have the responsibility to maintain a good input manual. If you make mistake on "DeleteUser", then the users can not acutally know the "TestUser1" has the right to execute "delete".
